halo2_proofs/src/fft.rs

```diff
@@ -0,0 +1,233 @@
+//! This is a module for dispatching between different FFT implementations at runtime based on environment variable `FFT`.
+
+use std::env::var;
+
+use ff::Field;
+
+use self::recursive::FFTData;
+use crate::{arithmetic::FftGroup, plonk::log_info};
+
+pub mod baseline;
+pub mod parallel;
+pub mod recursive;
+
+/// Read Environment Variable `FFT_MODE`
+pub fn get_fft_mode() -> usize {
+    var("FFT_MODE")
+        .unwrap_or_else(|_| "1".to_string())
+        .parse()
+        .expect("Cannot parse FFT_MODE env var as usize")
+}
+
+/// Read Environment Variable `DEGREE`
+#[allow(dead_code)]
+pub fn get_degree() -> usize {
+    var("DEGREE")
+        .unwrap_or_else(|_| "22".to_string())
+        .parse()
+        .expect("Cannot parse DEGREE env var as usize")
+}
```

Comment:
`get_fft_mode` and `get_degree` can simply be removed I think.

Reply:
`get_fft_mode` is gone.
`get_degree` is still used in some tests, so I moved it to the FFT test module.